specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -2691,34 +2929,45 @@
             "type": "string"
           },
           {
-            "name": "certificateName",
-            "description": "The integration account certificate name.",
-            "in": "path",
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "listKeyVaultKeys",
+            "description": "The key vault parameters.",
+            "in": "body",
             "required": true,
-            "type": "string"
+            "schema": {
+              "$ref": "#/definitions/ListKeyVaultKeysDefinition"
+            }
           },
           {
-            "$ref": "#/parameters/api-version"
+            "name": "skipToken",
+            "description": "The skip token.",
```

Comment:
skiptoken is defined as both a query parameter and an attribute of ListKeyVaultKeysDefinition. Are both required? Only the model attribute appears to be used in the example.

Reply:
You are correct, the model is the correct spot for it. I misread the example and added the extra one. It has been removed